src/queries/consul/uploads/upload.js

```diff
@@ -0,0 +1,49 @@
+import * as FileSystem from 'expo-file-system';
+
+import { namespace, secrets } from '../../../config';
+import { getConsulAuthToken } from '../../../helpers';
+
+// https://docs.expo.io/versions/latest/sdk/filesystem/#filesystemuploadasyncurl-fileuri-options
+export const uploadAttachment = async (uri, resourceRelation, resourceType) => {
```

Comment:
we could do `(uri, resourceRelation, resourceType = 'Proposal')` and avoid passing it with calling `await uploadAttachment(newProposalData.image, 'image');`, because it will be also `'Proposal'` for documents. if we somewhen need something different, we could pass a third param.